server/server.go

```diff
@@ -193,6 +194,12 @@ func (server HTTPServer) readReportForCluster(writer http.ResponseWriter, reques
 	}
 }
 
+// serveAPISpecFile serves an OpenAPI specifications file specified in config file
+func (server HTTPServer) serveAPISpecFile(writer http.ResponseWriter, request *http.Request) {
+	absPath, _ := filepath.Abs(server.Config.APISpecFile)
```

Comment:
well the file existence was tested before, but pls. add err checks there as well

Reply:
fixed